integration/update_field_compat_test.go

```diff
@@ -481,6 +481,97 @@ func TestUpdateFieldCompatMin(t *testing.T) {
 	testUpdateCompat(t, testCases)
 }
 
+func TestUpdateFieldCompatRename(t *testing.T) {
+	testCases := map[string]updateCompatTestCase{
+		"Simple": {
+			update: bson.D{{"$rename", bson.D{{"v", "foo"}}}},
+		},
+		"DuplicateField": {
+			update:     bson.D{{"$rename", bson.D{{"v", "v"}}}},
+			resultType: emptyResult,
+		},
+		"NonExistingField": {
+			update:     bson.D{{"$rename", bson.D{{"foo", "bar"}}}},
+			resultType: emptyResult,
+		},
+		"EmptyField": {
+			update:     bson.D{{"$rename", bson.D{{"", "v"}}}},
+			resultType: emptyResult,
+		},
+		"EmptyDest": {
+			update:     bson.D{{"$rename", bson.D{{"v", ""}}}},
+			resultType: emptyResult,
+		},
+		"DotDocumentMove": {
+			update:        bson.D{{"$rename", bson.D{{"v.foo", "boo"}}}},
+			skipForTigris: "schema violation",
+		},
+		"DotDocumentDuplicate": {
+			update:        bson.D{{"$rename", bson.D{{"v.foo", "v.array"}}}},
+			skipForTigris: "schema violation",
+		},
+		"DotDocumentNonExisting": {
+			update:     bson.D{{"$rename", bson.D{{"foo.bar", ""}}}},
+			resultType: emptyResult,
+		},
+		"DotArrayField": {
+			update:     bson.D{{"$rename", bson.D{{"v.array.0", ""}}}},
+			resultType: emptyResult,
+		},
+		"Multiple": {
+			update:        bson.D{{"$rename", bson.D{{"v.foo", "v.bar"}, {"v.42", "v.43"}}}},
+			skipForTigris: "schema violation",
+		},
+		"MultipleConflictDestSource": {
+			update: bson.D{{"$rename", bson.D{{"v", "foo"}, {"foo", "bar"}}}},
+			skip:   "Updating the path 'foo' would create a conflict at 'foo'",
+		},
+		"MultipleConflictDestFields": {
+			update:     bson.D{{"$rename", bson.D{{"v", "foo"}, {"v", "bar"}}}},
+			resultType: emptyResult,
+		},
+		"DotArrayNonExisting": {
+			update:     bson.D{{"$rename", bson.D{{"foo.0.baz", int32(1)}}}},
+			resultType: emptyResult,
+		},
+		"FieldEmpty": {
+			update:     bson.D{{"$rename", bson.D{}}},
+			resultType: emptyResult,
+		},
+		"FieldDoc": {
+			update:     bson.D{{"$rename", bson.D{{"v", primitive.D{}}}}},
+			resultType: emptyResult,
+		},
+		"FieldArray": {
+			update:     bson.D{{"$rename", bson.D{{"v", primitive.A{}}}}},
+			resultType: emptyResult,
+		},
+		"FieldNaN": {
+			update:     bson.D{{"$rename", bson.D{{"v", math.NaN()}}}},
+			resultType: emptyResult,
+			skip:       "CommandError returned",
```

Comment:
It we skip because it's a wrong response from FerretDB, we should create an issue.